definition_of_done.md

```diff
@@ -0,0 +1,38 @@
+# Definition of done
+
+This document clarifies, when an product backlog item is done. During scrum development process this gives us a checklist,
+what needs to be done.
+The individual points are a basis for acceptance criteria and are shortened or appended by specific measures as needed.
+
+## Documentation
+* [ ] Public api
+* [ ] API diagrams
+* [ ] [Tutorials](https://wiki.ros.org/pilz_robots/Tutorials/)
+* [ ] Readme
+* [ ] ROS wiki
+* [ ] Code (sphinx/doxygen)
+* [ ] Good commit messages ([example](https://dev.to/jacobherrington/how-to-write-useful-commit-messages-my-commit-message-template-20n9
+))
+* [ ] Changelog.md updated
+* [ ] Copyright headers
+
+## Reviews
+* [ ] Soft- and hardware architecture
+* [ ] Code (coding rules, style guide)
+* [ ] Test review (test plan and individual test cases)
+* [ ] Documentation review
+
+## Quality
+* [ ] CI pass:
+The travis job (for example in [pilz_robots](https://github.com/PilzDE/pilz_robots/blob/melodic-devel/.travis.yml)) checks full code coverage, successful test execution and code formatting using catkin_lint.
```

Comment:
Not sure if mentioning catkin_lint is necessary here. CI checks code formatting no matter the tool (catkin_lint, clang-format, etc.) right?